user_docs/en/userGuide.t2t

```diff
@@ -842,6 +843,22 @@ If you want to use these while still being able to use your cursor keys to read
 To toggle single letter navigation on and off for the current document, press NVDA+shift+space.
 %kc:endInclude
 
++++ Text paragraph navigation command +++[TextNavigationCommand]
+
+As mentioned above, you can jupm to next or previous text paragraph by pressing ``p`` or ``shift+p``. Here we define text paragraph to be a paragraph that appears to be written in complete sentences. This can be useful to find the beginning of readable content on various webpages, such as:

```

Comment:
```suggestion
You can jump to the next or previous text paragraph by pressing ``p`` or ``shift+p``.
Text paragraphs are defined by a group of text that appears to be written in complete sentences.
This can be useful to find the beginning of readable content on various webpages, such as:
```